src/ansys/fluent/core/launcher/watchdog_exec

```diff
@@ -143,25 +137,6 @@ if __name__ == "__main__":
                     logger.info("Exit call succeeded.")
             else:
                 logger.info("Fluent client not healthy.")
-
-        if fluent.connection_properties.inside_container:
-            logger.info(
-                "Running Fluent cleanup scripts inside container if they are still available..."
-            )
-            fluent.force_exit()
-            response = timeout_loop(
-                get_container,
-                IDLE_PERIOD * 3,
-                args=(cortex_host,),
-                expected="falsy",
-            )
-            if response:
-                logger.info(
-                    "Fluent container still alive somehow, directly terminating it..."
-                )
-                subprocess.run(["docker", "kill", cortex_host])
-            else:
-                logger.info("Fluent container successfully shut down.")
```

Comment:
`exit` method of container object is already being called inside force_exit() of both `FluentConnection` and `Session` hence removed from here.